package.xml

```diff
@@ -8,8 +8,14 @@
   <license>Apache2</license>
 
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
-  <exec_depend>xacro</exec_depend>
-  <!-- <exec_depend>pacmod3</exec_depend> --> <!-- not released for the Foxy yet -->
+
+  <depend>xacro</depend>
+
+  <exec_depend>kvaser_interface</exec_depend>
+  <exec_depend>pacmod3</exec_depend>
+  <exec_depend>pacmod_interface</exec_depend>
+  <exec_depend>raw_vehicle_cmd_converter</exec_depend>
+  <!-- <exec_depend>accel_brake_map_calibrator</exec_depend> -->
```

Comment:
Why is this commented out?

Reply:
I fixed it in https://github.com/tier4/lexus_description.iv/pull/24/commits/0f263806774284fb1dd6615f8a69aec47f14ee50